src/redux/infoModalSlice.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { RootState } from "./store";
+
+const initialState = {
+  toggle: false,
+  title: "",
+  text: "",
+};
+
+const infoModalSlice = createSlice({
+  name: "successModal",
+  initialState,
+  reducers: {
+    handleInfoToggle(state) {
+      state.toggle = !state.toggle;
+    },
+    setInfoTitle(state, action: { payload: string }) {
+      state.title = action.payload;
+    },
+    setInfoText(state, action: { payload: string }) {
+      state.text = action.payload;
+    },
+  },
+});
+
+export const selectInfoToggle = (state: RootState) => state.infoModal.toggle;
+export const selectInfoTitle = (state: RootState) => state.infoModal.title;
+export const selectInfoText = (state: RootState) => state.infoModal.text;
+
+export const { handleInfoToggle, setInfoTitle, setInfoText } =
+  infoModalSlice.actions;
+
+export default infoModalSlice.reducer;
```

Comment:
you don't need this slice, we do not store value like that in redux

Reply:
but how can i display `infoModal` module without redux? Don't I need to store something global to know if the request was successful or failed?

Reply:
There are so many ways to solve this problem either through react or toolkit. You can leave it as is, but for the future, such data is usually not stored in redux